sdks/java/extensions/google-cloud-platform-core/src/main/java/org/apache/beam/sdk/extensions/gcp/options/GcsOptions.java

```diff
@@ -49,7 +50,7 @@ public interface GcsOptions extends ApplicationNameOptions, GcpOptions, Pipeline
   @JsonIgnore
   @Description(
       "The GoogleCloudStorageReadOptions instance that should be used to read from Google Cloud Storage.")
-  @Default.InstanceFactory(GcsUtil.GcsReadOptionsFactory.class)
+  @Default.InstanceFactory(GcsUtilLegacy.GcsReadOptionsFactory.class)
```

Comment:
How about "GcsUtilV1"? this follows other branched classes naming convention, and in case they both exist in Beam code base for extended time and a "GcsUtilV3" appear in the future (hopefuly won't be the case).